make.py

```diff
@@ -0,0 +1,159 @@
+        extra_env={"POETRY_LIBPARSEC_BUILD_PROFILE": "ci"},
+    ),
+    # Flags used in poetry's `build.py` when command is `python-ci-build`
+    ("python-ci-libparsec-cargo-flags",): Cmd(
+        cmd=f"echo {PYTHON_CI_CARGO_FLAGS}",
+    ),
+    # Flags used in poetry's `build.py` when generating the release wheel
+    ("python-release-libparsec-cargo-flags",): Cmd(
+        cmd=f"echo {PYTHON_RELEASE_CARGO_FLAGS}",
+    ),
+    # Flags used in poetry's `build.py` when generating the dev wheel
+    ("python-dev-libparsec-cargo-flags",): Cmd(
+        cmd=f"echo {PYTHON_DEV_CARGO_FLAGS}",
+    ),
+}
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description=textwrap.dedent(
+            """\
+            Tired of remembering multiple sily commands ? Now here is a single sily command to remember !
+
+            Examples:
+            python make.py install  # Initial installation
+        """
+        ),
+    )
+    parser.add_argument(
+        "--quiet", "-q", action="store_true", help="No display, no color, no kitty, no hapiness"
+    )
+    parser.add_argument("--pretend", action="store_true", help="Don't actually run, just display")
+    parser.add_argument("command", help="The command to run", nargs="?")
+
+    # Handle `-- <extra_cmd_args>` in argv
+    has_reached_cmd_extra_args = False
+    extra_cmd_args = []
+    argv = []
+    for arg in sys.argv[1:]:
+        if has_reached_cmd_extra_args:
+            extra_cmd_args.append(arg)
+        elif arg == "--":
+            has_reached_cmd_extra_args = True
+        else:
+            argv.append(arg)
+
+    args = parser.parse_args(argv)
+    if not args.command:
+        print("Available commands:\n")
+        for aliases, cmds in COMMANDS.items():
+            print(f"{BOLD_YELLOW}{', '.join(aliases)}{NO_COLOR}")
+            cmds = (cmds,) if isinstance(cmds, Cmd) else cmds
+            display_cmds = [cmd.display() for cmd in cmds]
+            join = f"{GREY}; and {NO_COLOR}" if "fish" in os.environ.get("SHELL", "") else " && "
+            print(f"\t{join.join(display_cmds)}\n")
+
+    else:
+        for aliases, cmds in COMMANDS.items():
+            if args.command in aliases:
+                cmds = (cmds,) if isinstance(cmds, Cmd) else cmds
+                break
+        else:
+            raise SystemExit(f"Unknown command alias `{args.command}`")
+
+        for cmd in cmds:
+            if not args.quiet:
+                # Flush is required to prevent mixing with the output of sub-command
+                print(
+                    f"{cmd.display(extra_cmd_args)}\n",
+                    flush=True,
+                )
+                try:
+                    print(f"{PINK}{random.choice(CUTENESS)}{NO_COLOR}", flush=True)
+                except UnicodeEncodeError:
+                    # Windows crappy term couldn't encode kitty unicode :'(
```

Comment:
It fails on both VSCode integrated term and on Windows terminal :(

Anyway this is an issue with [cp1252 encoding](https://fr.wikipedia.org/wiki/Windows-1252) which cannot represent Unicode. So I'd rather skip this unneeded than have the comment entirely breaking because an exotic config twist is needed to change terminal encoding.

Reply:
:shrug: sad